### DISCUSSION TITLE
3 Collaborators for Private Repositories

### DISCUSSION THREAD

**Discussion by MrLinDowsMac** [community/community#22997] [New to GitHub]
<p>Github just announced unlimited private repositories, and up to 3 colaborators. But I don’t know exactly what it means with that. Just 3 collaborators for all my private repos? Or 3 collaborators per private repository?</p>

**Comment by Wabri** [comment, marked as answer]
<p>Every single private repository can have the limit of 3 collaborators.</p>

**Comment by ghost** [comment]
<p>Hey buddy, I do not know who you are. I am a newbie, I just joined GitHub. I want to start my FIRST Private Repository. I have no clue how to go about doing this? Can you help me?</p>

**Comment by ghost** [comment]
<p>Hey <a class="mention" href="/gitnick9">@gitnick9</a>,</p>
<p>Do you still need any help here? I saw your comment over <a href="https://github.community/t/a-crazy-newbie-wants-to-to-become-qualified-to-do-openai-in-3-years/1640/2">here</a>, so it looks like you figured it out!</p>

**Comment by cendekia** [comment]
<p>Are the max 3 collaborators mean also include the owner in it?</p>

**Comment by prachinegi** [comment]
<p>3 collaborators excluding owner.</p>
